Bottom: fce84d7d696b83c47a7104aa843f1af93f9fad36
Top:    f1e53b4bbeb4ed7d6b6581ab4c0e3ce0b67ee9e9
Author: Marina Samuel <msamuel@mozilla.com>
Date:   2018-05-11 16:02:26 -0400

Closes #396: Integration with Flower.


---
diff --git a/docker-compose.yml b/docker-compose.yml
index 8bd2c4dc..d9421385 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -45,3 +45,13 @@ services:
       - "15432:5432"
     command: "postgres -c fsync=off -c full_page_writes=off -c synchronous_commit=OFF"
     restart: unless-stopped
+  flower:
+    image: mher/flower:latest
+    command: flower
+    environment:
+      CELERY_BROKER_URL: redis://redis:6379/0
+      CELERY_RESULT_BACKEND: redis://redis:6379/0
+    ports:
+      - "5555:5555"
+    links:
+      - redis
